LoginDemo/LearningSelenium/Shopping.cs

```diff
@@ -0,0 +1,139 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using LearningSelenium.PageModel;
+using OpenQA.Selenium.Support.UI;
+using System.Threading;
+
+namespace LearningSelenium
+{
+    internal class Shopping
+    {
+        WebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
+            driver.Manage().Window.Maximize();
+
+        }
+
+        [Test]
+        public void TestLoginFail()
+        {
+            Login lg = new Login(driver);
+            driver.Navigate().GoToUrl("https://opensource-demo.orangehrmlive.com/web/index.php/auth/login");
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(20);
```

Comment:
Create an utility class to work with waitings

Reply:
it will be implemented soon